compiler/codegen/OMRInstruction.hpp

```diff
@@ -218,6 +218,9 @@ class OMR_EXTENSIBLE Instruction
    bool needsAOTRelocation() { return (_index & TO_MASK(NeedsAOTRelocation)) != 0; }
    void setNeedsAOTRelocation(bool v = true) { v ? _index |= TO_MASK(NeedsAOTRelocation) : _index &= ~TO_MASK(NeedsAOTRelocation); }
 
+   bool isLastWarmInstruction() { return (_index & TO_MASK(LastWarmInstruction)) != 0; }
+   void setLastWarmInstruction(bool v = true) { v ? _index |= TO_MASK(LastWarmInstruction) : _index &= ~TO_MASK(LastWarmInstruction); }
```

Comment:
Where is LastWarmInstruction ever set to something non-NULL?  I can't seem to find any code in this PR that does this.

Reply:
This feature requires support in a sub-project. It has to identify last warm instruction during instruction selection.

I put it in the PR description: 
- identify last warm instruction during instruction selection (currently, only in openj9)

Reply:
If this flag is only useful for a downstream project then it is possible to introduce this enum value in that project specifically.

The concern I have with introducing it in OMR without any references is that someone may innocuously use it to (seemingly) find the last instruction warm instruction in the method and never find it (because nothing ever sets it).  If this does live in OMR then I think more documentation is necessary.  I do have questions over its utilization, though.  Will this flag be updated if new instructions are inserted after the current warm instruction, or is it only intended to be used very late in code generation when no more instructions will be added.  Again, documentation will help here.

Reply:
The flag is used in OMR but can be set in a downstream project. If it is not set, OMR will not split warm and cold blocks. I think it's up to the code that sets it to maintain its correct value. I will try to document a bit more.

Reply:
Added comments.